src/shared/components/query/GeneSetSelector.tsx

```diff
@@ -45,7 +45,7 @@ export default class GeneSetSelector extends QueryStoreComponent<GeneSetSelector
 				label: 'User-defined List',
 				value: ''
 			},
-			...gene_lists.map(item => ({
+			...AppConfig.querySetsOfGenes == undefined ? [{label:"", value:""}] : AppConfig.querySetsOfGenes.map(item => ({
```

Comment:
why not `AppConfig.querySetsOfGenes?`

Reply:
You can take a look here: https://reactjs.org/docs/jsx-in-depth.html#spread-attributes